src/critics/prefer_forward_critic.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 FastSense, Samsung Research
+
+#include "mppic/critics/prefer_forward_critic.hpp"
+
+namespace mppi::critics
+{
+
+void PreferForwardCritic::initialize()
+{
+  auto node = parent_.lock();
+
+  auto getParam = utils::getParamGetter(node, name_);
+  getParam(power_, "prefer_forward_cost_power", 1);
+  getParam(weight_, "prefer_forward_cost_weight", 10.0);
+
+  RCLCPP_INFO(
+    logger_, "PreferForwardCritic instantiated with %d power and %f weight.", power_, weight_);
+}
+
+void PreferForwardCritic::score(
+  const geometry_msgs::msg::PoseStamped & /* robot_pos */, const xt::xtensor<double, 3> & trajectories,
+  const xt::xtensor<double, 2> & /* path */, xt::xtensor<double, 1> & costs,
+  nav2_core::GoalChecker * /* goal_checker */)
+{
+  using namespace xt::placeholders;
```

Comment:
```suggestion
  using namespace xt::placeholders;  // NOLINT
```
Required for linting on using namespaces